eos-upgrade-eos2-to-eos3

```diff
@@ -199,6 +194,36 @@ gpg-verify-summary=false
 url=file:///var/lib/flatpak-external-apps
 EOF
 
+if [[ $(blkid -t LABEL=extra) ]]; then
```

Comment:
other scripts that look to identify this setup just check for the existence of /dev/disk/by-label/extra